devicetypes/smartthings/ikea-button.src/ikea-button.groovy

```diff
@@ -171,10 +172,13 @@ def configure() {
 	log.debug "Configuring device ${device.getDataValue("model")}"
 
 	def cmds = zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x21, DataType.UINT8, 30, 21600, 0x01) +
-			zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x21) +
```

Comment:
I would do something like this:

```
def cmds = []

if (!isSomfySituo()) {
	cmds += zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x21, DataType.UINT8, 30, 21600, 0x01) +
		zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x21)
}

cmds += zigbee.addBinding(zigbee.ONOFF_CLUSTER) +
	readDeviceBindingTable() // Need to read the binding table to see what group it's using
```

No need to configure reporting on a cluster that isn't supported.

Reply:
When it comes to Somfy Situo something unclear happens. When the POWER_CONFIGURATION_CLUSTER is not configured then there are no reports coming from the ON_OFF_CLUSTER. 

We've supposed that it might be caused by the fact that the device needs to get any kind of configure command to start reporting and we've tried with adding zigbee.onOffConfig() but it did not help.

Reply:
@BJanuszS I would suggest reaching out to the manufacturer in this case.